interop/test_utils.go

```diff
@@ -454,6 +454,41 @@ func DoCancelAfterFirstResponse(tc testpb.TestServiceClient) {
 	}
 }
 
+// DoStatusCodeAndMessage checks that the status code is propagated back to the client.
+func DoStatusCodeAndMessage(tc testpb.TestServiceClient) {
+	var code int32 = 2
+	msg := "test status message"
+	expectedErr := grpc.Errorf(codes.Code(code), msg)
+	respStatus := &testpb.EchoStatus{
+		Code:    proto.Int32(code),
+		Message: proto.String(msg),
+	}
+	// Test UnaryCall.
+	req := &testpb.SimpleRequest{
+		ResponseStatus: respStatus,
+	}
+	if _, err := tc.UnaryCall(context.Background(), req); err.Error() != expectedErr.Error() {
+		grpclog.Fatalf("%v.UnaryCall(_, %v) = _, %v, want _, %v", tc, req, err, expectedErr)
+	}
+	// Test FullDuplexCall.
+	stream, err := tc.FullDuplexCall(context.Background())
+	if err != nil {
+		grpclog.Fatalf("%v.FullDuplexCall(_) = _, %v, want <nil>", tc, err)
+	}
+	stream_req := &testpb.StreamingOutputCallRequest{
```

Comment:
go vet is complaining about this line
don't use underscores in Go names; var stream_req should be streamReq